server/libs/cache/userCache.js

```diff
@@ -0,0 +1,25 @@
+import { redis } from '../redis'
+
+export default {
+  get(req, callback) {
+    const userId = req.json.userId
+    redis.get(userId, (err, data) => {
```

Comment:
Instead of just keying this as userId, it would be better to do something like `scitran:user:${userId}` to prevent collisions and allow for future clearing of parts of the cache.

Reply:
changed!